src/components/tl/mlx5/mcast/tl_mlx5_mcast_helper.h

```diff
@@ -0,0 +1,322 @@
+/**
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "tl_mlx5.h"
+
+#ifndef TL_MLX5_MCAST_HELPER_H_
+#define TL_MLX5_MCAST_HELPER_H_
+#include "tl_mlx5_mcast_progress.h"
+#include "utils/ucc_math.h"
+
+/* this function returns the number of signaled sends that
+ * have been completed or -1 in case of error */
+static inline int ucc_tl_mlx5_mcast_poll_send(ucc_tl_mlx5_mcast_coll_comm_t *comm)
```

Comment:
this function returns number completed sends, but you don't use this number and only check if it's negative. In that case it's better to return status